sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -24,7 +24,11 @@ class CommandGraphTest : public ::testing::Test {
 public:
   CommandGraphTest()
       : Mock{}, Plat{Mock.getPlatform()}, Dev{Plat.get_devices()[0]},
-        Queue{Dev}, Graph{Queue.get_context(), Dev} {}
+        Queue{Dev}, Graph{Queue.get_context(), Dev} {
+    // We need to disable backend for unitests as backends are not loaded
+    auto GraphImpl = sycl::detail::getSyclObjImpl(Graph);
+    GraphImpl->setEmulationModeForced(true);
```

Comment:
This feels a little hacky to me, is there some reason we can't add the PI command buffer calls to the mocking for the unit tests?

Reply:
According to my understanding, Unitests are not supposed to need an actual device to run, so I suppose it is why backend plugins are not linked to the Unitest Mock. And it is why I used this little trick.